media/css/firefox/os/firefox-os-new.less

```diff
@@ -0,0 +1,960 @@
+@import "../../sandstone/lib.less";
+
+* {
+    .border-box();
+}
+#outer-wrapper {
+    background: #0c99d5 url(/media/img/firefox/os-new/raysoflight.png) 0 0 no-repeat;
+}
+#wrapper {
+    background-color: #0C99D5;
+    width: 100%;
+    padding-bottom: 0;
+}
+main {
+    margin: 0 auto;
+    text-align: center;
+    h1 {
+        background: transparent url(/media/img/firefox/os-new/fxos_wordmark420.png) 0 0 no-repeat;
+        margin: 0px auto 20px auto;
```

Comment:
For any `20px` values (and often `10px`, `30px`, and `40px`), we usually use the `@baseLine` variable. Not critical for launch.